src/traces/bar/layout_attributes.js

```diff
@@ -63,5 +63,17 @@ module.exports = {
             'Sets the gap (in plot fraction) between bars of',
             'the same location coordinate.'
         ].join(' ')
-    }
+    },
+    displaytotal: {
```

Comment:
`displaytotal` is too general while this attribute lives in `layout`.
How about `barshowtotal`?

Reply:
Much better indeed.
Should I also rename `totaltemplate` to `bartotaltemplate`, so that each bar specific option starts with `bar` ? Or do we want to reuse `totaltemplate` for area charts as well ?